src/App.tsx

```diff
@@ -1,24 +1,353 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+type PartialTodo = Omit<Todo, 'id'>;
+
+const USER_ID = 11880;
+
+enum FilterBy {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
redundant to make state for filtteredTodos.
make function which wiill except allTodos and filterBy and return filtered todos.